app/notifications/invisiblize_developer_notification.rb

```diff
@@ -0,0 +1,18 @@
+class InvisiblizeDeveloperNotification < Noticed::Base
+  deliver_by :database
+  deliver_by :email, mailer: "DeveloperMailer", method: :flagged
+
+  param :developer
+
+  def developer_profile_url
+    developer_path(developer)
+  end
+
+  def home_url
+    root_path
+  end
```

Comment:
What are these two methods used for?

Reply:
Nothing now! Sorry about that, I had them in for something else.